app/controllers/api/v0/users_controller.rb

```diff
@@ -10,7 +10,7 @@ class UsersController < ApiController
       ].freeze
 
       def show
-        relation = User.select(SHOW_ATTRIBUTES_FOR_SERIALIZATION)
+        relation = User.joins(:profile).select(SHOW_ATTRIBUTES_FOR_SERIALIZATION)
```

Comment:
I'm starting to think if we should consider (not in this PR), eager loading profiles using `strict_loading: true` see https://blog.saeloun.com/2020/02/25/rails-strict-loading-mode-to-fix-n-1.html - we would need to understand if the profile is always or very often needed when we load the user object

Reply:
Yes, I have a comment to this effect somewhere. Not explicitly about strict loading, but about figuring out the answer to this issue so we can decide on the best strategy :-)